contracts/base/strategies/multi/AaveMaiBalStrategyBase.sol

```diff
@@ -12,31 +12,31 @@
 
 pragma solidity 0.8.4;
 
-import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "./../../../openzeppelin/SafeERC20.sol";
 import "./../../../third_party/IERC20Extended.sol";
-import "./../StrategyBase.sol";
+import "./../ProxyStrategyBase.sol";
 import "./pipelines/LinearPipeline.sol";
 import "../../interface/strategies/IMaiStablecoinPipe.sol";
 import "../../interface/strategies/IAaveMaiBalStrategyBase.sol";
 
 /// @title AAVE->MAI->BAL Multi Strategy
 /// @author belbix, bogdoslav
-contract AaveMaiBalStrategyBase is StrategyBase, LinearPipeline, IAaveMaiBalStrategyBase {
+contract AaveMaiBalStrategyBase is ProxyStrategyBase, LinearPipeline, IAaveMaiBalStrategyBase {
   using SafeERC20 for IERC20;
 
   /// @notice Strategy type for statistical purposes
   string public constant override STRATEGY_NAME = "AaveMaiBalStrategyBase";
   /// @notice Version of the contract
   /// @dev Should be incremented when contract changed
-  string public constant VERSION = "1.2.0";
+  string public constant VERSION = "2.0.0";
   /// @dev Placeholder, for non full buyback need to implement liquidation
   uint256 private constant _BUY_BACK_RATIO = 10000;
 
   /// @dev Assets should reflect underlying tokens for investing
   address[] private _assets;
 
   // cached total amount in underlying tokens, updated after each deposit, withdraw and hardwork
-  uint256 private _totalAmount = 0;
+  uint256 public _totalAmount = 0;
```

Comment:
is it necessary?
underscore means private variable
but without underscore totalAmount so popular :)

Reply:
Renamed to totalAmount. Public varibales much better for invistigation.
Also I would like to make _maiStablecoinPipe and _maiCamPipe. What do you think?
Did you tested gas consumption of usage public and internal / private variables?